be/src/storage/task/engine_batch_load_task.cpp

```diff
@@ -181,11 +208,14 @@ Status EngineBatchLoadTask::_push(const TPushReq& request, std::vector<TTabletIn
     Status res;
 
     if (request.tablet_type == TTabletType::TABLET_TYPE_LAKE) {
+        auto tablet_id = request.tablet_id;
+        auto tablet_version = (request.version == -1) ? choose_any_version(tablet_id) : request.version;
```

Comment:
1. why any version will work for it?
2. when will the request.version == -1, haven't checked the detail from FE side.

Reply:
1. In short, if the version of FE is less than 3.3.0, then there will be no fast schema evolution and any version will contain the same tablet schema.
2. request.version will alwasy be -1 before 3.3.0, it's an unused field before 3.3.0.